### PR TITLE
[fix] #3546: Fix time trigger tests get stuck

### LINKED ISSUE #3546: [bug]: the test `pre_commit_trigger_should_be_executed` get stuck sometimes

### PR DESCRIPTION
**Pull request by Erigara**
## Description

Change order in which we submit first transaction and start listening for events.

### Linked issue

Closes #3546 

### Benefits

Test doesn't get stuck.

### Checklist

- [x] I've read `CONTRIBUTING.md`
- [x] I've used the standard signed-off commit format (or will squash just before merging)
- [x] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

**Review by pesterev — APPROVED** [review]

File: client/tests/integration/triggers/time_trigger.rs [hunk 1]
```diff
@@ -34,6 +34,9 @@ fn time_trigger_execution_count_error_should_be_less_than_15_percent() -> Result
     wait_for_genesis_committed(&vec![test_client.clone()], 0);
     let start_time = current_time();
 
+    // Start listening BEFORE submitting any transaction not to miss any block committed event
+    let event_listener = get_block_committed_event_listener(&test_client)?;
```
Comment (Arjentix) [inline client/tests/integration/triggers/time_trigger.rs:38]: So this test is ignored, but you fixed it too? Interesting. Mabye we should unignore it if it works fine now?

File: client/tests/integration/triggers/time_trigger.rs [hunk 2]
```diff
@@ -178,6 +181,9 @@ fn mint_nft_for_every_user_every_1_sec() -> Result<()> {
     let (_rt, _peer, mut test_client) = <PeerBuilder>::new().with_port(10_780).start_with_runtime();
     wait_for_genesis_committed(&vec![test_client.clone()], 0);
 
+    // Start listening BEFORE submitting any transaction not to miss any block committed event
+    let event_listener = get_block_committed_event_listener(&test_client)?;
```
Comment (Arjentix) [inline client/tests/integration/triggers/time_trigger.rs:185]: Shouldn't we create this just before the last `submit()` before submitting sample instructions?

I mean shouldn't we put this before line 240 (in current diff)

File: client/tests/integration/triggers/time_trigger.rs (hunk 1 above)
Comment (appetrosyan) [reply client/tests/integration/triggers/time_trigger.rs:38]: The whole point is to unignore these tests. We only remove the tests that are redundant after the Pytest 
sdk was merged

File: client/tests/integration/triggers/time_trigger.rs (hunk 2 above)
Comment (Erigara) [reply client/tests/integration/triggers/time_trigger.rs:185]: For what? there is no transactions submitted between this two points and iroha don't produce empty blocks so effect should be the same

Comment (Erigara) [reply client/tests/integration/triggers/time_trigger.rs:185]: Yes, indeed

**Review by pesterev — APPROVED** [review]

**Pull request merged**